app/models/post.rb

```diff
@@ -1,3 +1,12 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :comments
+
+  def update_post
+    user.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
+  end
```

Comment:
- [ ] this is the same issue as when you're getting the 3 most recent post, you should first order the comments and after that, you should set the limit for the number of comments.